Vagrantfile

```diff
@@ -17,6 +17,11 @@ Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
     cfg.vm.provision "shell", path: "scripts/create-machine.ps1", args: "-machineHome #{home} -machineName 2016"
   end
 
+  config.vm.define "2016-box" do |cfg|
+    cfg.vm.box     = "StefanScherer/windows_2016_docker"
+    cfg.vm.provision "shell", path: "scripts/create-machine.ps1", args: "-machineHome #{home} -machineName 2016"
```

Comment:
Can you change the args to `-machineName 2016-box` so the vagrant vm name matches the docker machine name?